simpleflow/swf/process/decider/base.py

```diff
@@ -233,3 +229,23 @@ def decide(self, decision_response, task_list):
             decisions = [decision]
 
         return decisions
+
+
+def process_decision(poller, decision_response):
+    logger.info("taking decision for workflow {}".format(poller._workflow_name))
```

Comment:
minor: rename `_workflow_name` to show it's not so private

Reply:
Sure